source/AI.cpp

```diff
@@ -789,6 +789,10 @@ void AI::Step(const PlayerInfo &player, Command &activeCommands)
 			}
 		}
 
+		// Update any orders NPCs may have been given by their associated mission.
```

Comment:
I dont think we usually put empty space before an else
no big deal but it does make it a bit less readable I think